packages/php-db-import-export/tests/functional/Snowflake/ToFinal/FullImportTest.php

```diff
@@ -33,6 +36,97 @@ protected function setUp(): void
         $this->createSchema($this->getDestinationSchemaName());
     }
 
+    /**
+     * Test is testing loading of semi-structured data into typed table.
+     *
+     * We ignore here GEOGRAPHY and GEOMETRY as they act differently when casting from string
```

Comment:
we ignore in this test... ale takze co se stane, kdyz to nekdo pouzije?

Reply:
Hehe upravil sem sem on ten GEOMETRY a GEOGRAPHY funguje automaticky :) Takže sem ho přidal do testu.